pkg/metricsservice/client.go

```diff
@@ -37,7 +37,8 @@ type GrpcClient struct {
 }
 
 func NewGrpcClient(url, certDir string) (*GrpcClient, error) {
-	retryPolicy := `{
+	defaultConfig := `{
+		"loadBalancingConfig": [{"round_robin":{}}],
```

Comment:
I'm not sure if here makes sense as this is used for calling from metrics server to operator and it'll have a single endpoint available because it depends on leader election API.

Reply:
thanks that's a good callout, removed since it's unnecessary